### DISCUSSION TITLE
It doesn't seem possible to use Using Fine-grained PATs to generate a repo across orgs

### DISCUSSION THREAD

**Discussion by GordonBeeming** [community/community#55144] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I'm trying to use Fine-grained PATs like a good person 😇 but it appears you can't have the PAT having read access to a source org/repo and then the correct permissions in the target org because the fine-grained permissions are scoped max to 1 resource owner.

[Create a repository using a template](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-using-a-template)

1. Am I missing something from the docs?

**Comment by byrneh** [comment]
@GordonBeeming no you are not missing something 
The documentation states

`Each token can only access resources owned by a single user or organization`.

**Reply by GordonBeeming** [reply]
I was hoping that there was undocumented params I could pass for target pat or anything else similar even if unsupported (for now) 🫣. It doesn't seem like there's anything in the works though 😕
